src/ZiraLink.Api.Application/ProjectService.cs

```diff
@@ -55,6 +59,9 @@ public async Task<Guid> CreateAsync(long customerId, string title, DomainType do
             if (isDomainExists)
                 throw new ApplicationException("Domain already exists");
 
+            if (!_tools.CheckDomainExists(internalUrl).Result)
+                throw new ApplicationException("Domain already exists");
```

Comment:
Message suggestion: Public domain is not allowed

Reply:
Done